go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -177,6 +177,7 @@ func newDefaultNetworkControllerCommon(cnci *CommonNetworkControllerInfo,
 		cnci.watchFactory.NodeCoreInformer().Lister(),
 		cnci.nbClient,
 		addressSetFactory,
+		DefaultNetworkControllerName,
```

Comment:
I'm still puzzled that we initially changed it to have its own controller name and now it's reverted back to the generic one. I thought the idea of having a unique name was to better manage the ownership of the entries in the DB. Anyway, perhaps this was discussed during a team meeting and I missed it.